docs/guides/integrating-the-feature-hub.md

```diff
@@ -46,9 +46,10 @@ const manager = new FeatureAppManager(registry);
 
 **Note:** The integrator needs a self-selected but unique consumer ID to
 register or [consume][consuming-feature-services] Feature Services (in the
-example above it is `'acme:integrator'`). All Feature services registered
-together via the `registerProvider` method are automatically sorted
-topologically and therefore do not have to be registered in the correct order.
+example above it is `'acme:integrator'`). All Feature Services registered
+**together** using the `registerProvider` method of the `FeatureServiceRegistry`
```

Comment:
Does highlighting this really explain this better? Or is this better documented by an additional sentence?

Reply:
ah just saw https://github.com/sinnerschrader/feature-hub/pull/218/files#diff-32a1ffa5a1854c40b3984366a7adcc13R9! Maybe link it from here?